tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -140,6 +140,20 @@ module.exports = {
         .waitForOutstandingAjaxCalls()
         .useCss()
     },
+    /**
+     * @param {string} path
+     */
+    selectFileOrFolder: function (path) {
+      const fileCheckbox = this.getFileRowSelectorByFileName(path) +
+        this.elements['checkboxInFileRow'].selector
+
+      return this.initAjaxCounters()
+        .waitForFileVisible(path)
+        .useXpath()
+        .click(fileCheckbox)
+        .waitForOutstandingAjaxCalls()
```

Comment:
are there ajax calls when selecting the file?

Reply:
No, there aren't 🤫